src/coreComponents/mainInterface/initialization.cpp

```diff
@@ -253,12 +263,112 @@ std::unique_ptr< CommandLineOptions > parseCommandLineOptions( int argc, char *
 }
 
 
+void outputVersionInfo()
+{
+#if defined(__clang__)
```

Comment:
```suggestion
#if defined(__INTEL_COMPILER)
  GEOSX_LOG_RANK_0( "  - Intel compiler version: " << __INTEL_COMPILER << " (" << __VERSION << ")" );
#endif
```

Reply:
I stopped trying to support intel compilers a few years back. Are they better these days? Is there a compelling reason to support them?

Reply:
> We stopped trying to support intel compilers

Didn't know about that...